build/commands/lib/util.js

```diff
@@ -722,6 +722,38 @@ const util = {
     return applyPatches()
   },
 
+  updateChromeVersion: () => {
+    const braveVersionParts = config.braveVersion.split('.')
+    assert(braveVersionParts.length == 3)
+
+    const versionFilePath = path.join(config.srcDir, 'chrome', 'VERSION')
+    const versionLines = fs.readFileSync(versionFilePath).toString().split('\n')
+    assert(versionLines.length >= 4)
+
+    const versionLineRegex = /^(\w+)=(\d+)$/
+    for (let line = 0; line < 4; ++line) {
+      assert(
+        versionLines[line].search(versionLineRegex) == 0,
+        `${versionLines[line]} (${line}) doesn't match ${versionLineRegex}`
+      )
+      if (line == 0) {
+        // Keep MAJOR.
+        assert(versionLines[line].startsWith('MAJOR='))
+      } else {
+        // Set MINOR, BUILD, PATCH to Brave version.
+        versionLines[line] = versionLines[line].replace(
+          versionLineRegex,
+          `$1=${braveVersionParts[line - 1]}`
+        )
+      }
+    }
+    fs.writeFileSync(versionFilePath, versionLines.join('\n'))
+    versionLines.pop()
+    console.log(
+      `chrome/VERSION: ${versionLines.join('.').replace(/[A-Z=]/g, '')}`
+    )
```

Comment:
Match logging style with rest of sync in order to differentiate status entries with output from child processes:
```js
Log.progress(`Updated chrome/VERSION: ${versionLines.join('.').replace(/[A-Z=]/g, '')}`)
```

Reply:
This is a chunk of work, just a small one. It's used to separate program output from progress in the sync process. There is also Log.status if you think that's more appropriate.